src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,22 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: do
+# TODO: remove any unnecessary queries / lines here
+#  - fix_hgnc_mappings
+#  - maybe make a comment about other ones i thought to delete
+$(COMPONENTSDIR)/icd11.owl: $(TMPDIR)/icd11_relevant_signature.txt | component-download-icd11.owl
```

Comment:
## Source has `isObsolete` and `owl:deprecated`
@matentzn @twhetzel I initially found `isObsolete` and was going to rename that to `owl:deprecated`. However, I now see that the source has both. Not sure what I intend to do yet. Perhaps there is some nuance difference between these two. I'm not sure if we care about that nuance and should do some retention of both of these properties, or if I should rename them all to `owl:deprecated`.

Perhaps one thing I can do is an analysis of the cases where a class is both `isObsolete` and `owl:deprecated`, and also look at classes that only have one of these properties.

Reply:
Can maybe write a SPARQL query or use OAK for the usage of each, or OAK. And see if there's an interesection.

Reply:
Gonna do this in the future. No longer in scope of this PR unless I hear otherwise:
- https://github.com/monarch-initiative/icd11/issues/9